composer.json

```diff
@@ -1,55 +1,52 @@
 {
-  "name": "temporal/open-telemetry-interceptors",
-  "description": "This package provides OpenTelemetry interceptors for Temporal PHP SDK",
-  "keywords": [
-    "temporal",
-    "open-telemetry",
-    "interceptors",
-    "php"
-  ],
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Pavel Buchnev (butschster)",
-      "email": "pavel.buchnev@spiralscout.com"
+    "name": "temporal/open-telemetry-interceptors",
+    "description": "This package provides OpenTelemetry interceptors for Temporal PHP SDK",
+    "keywords": [
+        "temporal",
+        "open-telemetry",
+        "interceptors",
+        "php"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Pavel Buchnev (butschster)",
+            "email": "pavel.buchnev@spiralscout.com"
+        },
+        {
+            "name": "Aleksei Gagarin (roxblnfk)",
+            "email": "alexey.gagarin@spiralscout.com"
+        }
+    ],
+    "homepage": "https://temporal.io/",
+    "support": {
+        "docs": "https://docs.temporal.io/",
+        "chat": "https://t.mp/slack"
     },
-    {
-      "name": "Aleksei Gagarin (roxblnfk)",
-      "email": "alexey.gagarin@spiralscout.com"
-    }
-  ],
-  "homepage": "https://temporal.io/",
-  "support": {
-    "docs": "https://docs.temporal.io/",
-    "chat": "https://t.mp/slack"
-  },
-  "require": {
-    "php": ">=8.1",
-    "grpc/grpc": "^1.42",
-    "open-telemetry/exporter-otlp": "^0.0.17",
-    "open-telemetry/sdk": "^0.0.17",
-    "symfony/http-client": "^6.2",
-    "nyholm/psr7": "^1.8",
-    "guzzlehttp/promises": "^1.5",
-    "php-http/message-factory": "^1.1",
-    "temporal/sdk": "dev-interceptors"
-  },
-  "require-dev": {
-    "phpunit/phpunit": "^10.0"
-  },
-  "autoload": {
-    "psr-4": {
-      "Temporal\\OpenTelemetry\\": "src"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Temporal\\OpenTelemetry\\Tests\\": "tests"
-    }
-  },
-  "config": {
-    "sort-packages": true
-  },
-  "minimum-stability": "dev",
-  "prefer-stable": true
+    "require": {
+        "php": ">=8.1",
+        "guzzlehttp/promises": "^2.0",
+        "nyholm/psr7": "^1.8",
```

Comment:
Can we replace it with `psr/http-message-implementation`?

Reply:
@msmakouz  After [bc85777](https://github.com/temporalio/sdk-php-interceptors-opentelemetry/pull/2/commits/bc85777f84e090bb1ad57e2372f876cbd0f181ef) it need to test changes on `temporal/samples-php`